scalafix/build.sbt

```diff
@@ -19,13 +23,20 @@ inThisBuild(
     scalacOptions ++= List(
       "-Yrangepos"
     ),
+    publishTo := {
+      val nexus = "https://oss.sonatype.org/"
+      if (isSnapshot.value)
+        Some("snapshots" at nexus + "content/repositories/snapshots")
+      else
+        Some("releases"  at nexus + "service/local/staging/deploy/maven2")
+    }
```

Comment:
This can go away if we integrate into the main repo, with sbt-ci-release.

If we don't, we should replace it with sbt-ci-release.